sql/parser/datum.go

```diff
@@ -333,10 +333,9 @@ func (d DBytes) String() string {
 	return encodeSQLBytes(string(d))
 }
 
-// DDate is the date Datum.
-type DDate struct {
-	time.Time // Must always be UTC!
-}
+// DDate is the date Datum represented as the number of days after
+// the Unix epoch.
+type DDate int64
```

Comment:
I can do that
On Tue, Oct 13, 2015 at 3:51 PM Vivek Menezes <vivek@cockroachlabs.com>
wrote:

> A struct is valuable if you need to embed an interface. Adding the int64
> to a struct will make the code uglier. Luckily in Go you have to explicitly
> cast so it's safe from implicit madness. I don't think this is an
> improvement.
>
> On Tue, Oct 13, 2015 at 3:45 PM Tamir Duberstein <notifications@github.com>
> wrote:
>
>> In sql/parser/datum.go
>> <https://github.com/cockroachdb/cockroach/pull/2811#discussion_r41914521>
>> :
>>
>> > @@ -333,10 +333,9 @@ func (d DBytes) String() string {
>> >  	return encodeSQLBytes(string(d))
>> >  }
>> >
>> > -// DDate is the date Datum.
>> > -type DDate struct {
>> > -	time.Time // Must always be UTC!
>> > -}
>> > +// DDate is the date Datum represented as the number of days after
>> > +// the Unix epoch.
>> > +type DDate int64
>>
>> possibly pedantic: do we want to make this a struct containing an int64
>> so that it can't just be cast?
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/cockroachdb/cockroach/pull/2811/files#r41914521>.
>>
>